### PR TITLE
Revert "Aws-mp-fix"

### PR DESCRIPTION
**Pull request by Joeper214**
Reverts skyarch-networks/skyhopper#340

Reason:

Sorry I made a Mistake. This branch shouldn't have merged into `develop` because this contains `aws-marketplace` feature that should not be in `develop. I'll revert this one.

### CODE REVIEW

**Comment by Joeper214** [comment]
すみません。　@root-beer .

**Comment by root-beer** [comment]
Don't worry.

File: Gemfile [hunk 1]
```diff
@@ -32,10 +32,6 @@ gem 'jbuilder'
 # Modularize javascript code in application
 gem 'browserify-rails'
 
-# Fix the version for Amazon Linux
-# http://stackoverflow.com/questions/22950020/multijson-adaptererror-rails-4-ruby-2-passenger
-gem 'multi_json', '1.7.8'
```
Comment (Joeper214) [inline Gemfile]: But I will still include this fix to be merged into `develop` branch. Is it okay?

**Comment by Joeper214** [comment]
`develop` != `aws-marketplace`

**Comment by Joeper214** [comment]
Please review and Merge PR. @root-beer 🙇

**Comment by root-beer** [comment]
I'll review it now.

**Comment by root-beer** [comment]
LGTM. Thanks!

**Comment by Joeper214** [comment]
ありがとうございます。:bow:

**Pull request merged**